src/Hosting/Hosting/src/WebHostBuilderExtensions.cs

```diff
@@ -86,7 +86,7 @@ public static IWebHostBuilder Configure(this IWebHostBuilder hostBuilder, Action
     /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/> to configure.</param>
     /// <param name="startupFactory">A delegate that specifies a factory for the startup class.</param>
     /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
-    /// <remarks>When using the il linker, all public methods of <typeparamref name="TStartup"/> are preserved. This should match the Startup type directly (and not a base type).</remarks>
+    /// <remarks>When in a trimmed app, all public methods of <typeparamref name="TStartup"/> are preserved. This should match the Startup type directly (and not a base type).</remarks>
```

Comment:
I'm not very familiar how these comments work for the aspnetcore repo in terms of what shows up in intellisense. Are triple slash comments the source of truth? If not, and if instead we use a model similar to runtime where we have api-docs being the source of truth, then perhaps we should make sure that these updates in comments get applied there too.

Reply:
Hmm, that's a good question. I assumed that the triple-slash comments were the source of truth because I've never seen anyone updating them anywhere else. Anyone else know?

Reply:
XML comments are the source of truth in dotnet/aspnetcore.